src/screens/login-id/index.tsx

```
@@ -0,0 +1,22 @@
+import React from "react";
+import Button from "@/common/Button";
+
+const LoginIdScreen: React.FC = () => {
+  return (
+    <div className="p-6 mx-auto my-4 max-w-md bg-backgroundWidget rounded-lg shadow-md border">
+      <h1 className="text-2xl font-bold mb-4">Enter Your Email</h1>
+      {/* Add email input field here later */}
+      <p className="mb-6 text-sm text-textSecondary dark:text-gray-400">
+        Please enter the email address associated with your account.
```

Should we demonstrate how to read `texts` from the SDK, and only fall back to hardcoded strings when `texts` is unavailable?

For example:

```tsx
<h1 className="text-2xl font-bold mb-4">
  {screenInstance.screen.texts?.title ?? 'Enter Your Email'}
</h1>
```

This pattern can be applied consistently across all hardcoded text elements.

---

we can ignore code in screens. This is in progress and we will update our mock data with least context_config. So `screen.texts` will be mostly irrelevant.